Flow.Launcher.Plugin/IPublicAPI.cs

```diff
@@ -26,11 +26,11 @@ public interface IPublicAPI
 
         /// <summary>
         /// Restart Flow Launcher
-        /// </summary>
+        /// </summar
```

Comment:
is this typo?

Reply:
yes. i fixed while i was updating the comments.